go/vt/vttablet/onlineddl/executor.go

```diff
@@ -1561,36 +1634,15 @@ func (e *Executor) CancelPendingMigrations(ctx context.Context, message string)
 }
 
 // scheduleNextMigration attemps to schedule a single migration to run next.
-// possibly there's no migrations to run. Possibly there's a migration running right now,
-// in which cases nothing happens.
+// possibly there's no migrations to run.
+// The effect of this function is to move a migration from 'queued' state to 'ready' state, is all.
+// Notice that the query sqlScheduleSingleMigration embeds some logic inside. We may choose
+// to refactor the logic into the app
```

Comment:
hmm.. which app is this referring to?

Reply:
Clarified, moved explanation into the function itself. This is more a developer's note for future me/us.